src/components/SuccessRateGauge.tsx

```diff
@@ -40,19 +40,55 @@ const getDisplayValue = (
   const color = getSuccessRateThresholdColor(thresholds, thresholdKey, successRateKey!);
 
   return {
-    title,
+    title: '',
     color: color,
     numeric: successRateKey || 0,
     text: successRate.noData ? 'N/A' : displayValueKey!,
   };
 };
 
-export const SuccessRateGauge = ({ title, type, id, labelNames, labelValues, height, width, onClick }: Props) => {
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    titleContainer: css`
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      gap: ${theme.spacing(1)};
+      height: 20px;
+      justify-content: flex-start;
+      margin-x: ${theme.spacing(1)};
+    `,
+    title: css`
+      padding-left: ${theme.spacing(2)};
+    `,
+    container: css`
+      display: flex;
+      flex-direction: column;
+      margin-top: ${theme.spacing(1)};
+      margin-x: ${theme.spacing(1)};
+      height: 100%;
+    `,
+    infoIcon: css`
+      cursor: default;
+    `,
+  };
+}
```

Comment:
As all of these are duplicated here and in the `LatencyGauge` I think it could be worth creating a new component instead which is effectively a wrapper around Grafana's `BigValue` and convert these two components to use that instead?